### PR TITLE
Chore: Prepare the Preptember 2024

### PR DESCRIPTION
**Pull request by adiati98**
## Description

This PR updates the repository to be ready for the Preptember 2024 with the following changes:

- Adds new issue templates
- Updates the Contributing file
- Updates the PR template
- Archives previous Preptember files

### CODE REVIEW

**Review by BekahHW — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,33 +10,46 @@ Welcome to the contributing guide for Virtual Coffee's Preptember repo! Thank yo
...
-## Contribution type
+## Contribution Type
 
 There are two types of contributions to this repo:
 
-1. 🌱 **Practicing Open Source** — Add your name, link to your GitHub profile, and your favorite coffee (or any other drink you like!) to the `preptember-participants.md` file.
+1. 🌱 **Practicing Open Source** — Add your name, link to your GitHub profile, and your favorite cake or snack to the `preptember-participants.md` file.
 
-2. 📃 **Repositories List** — Add the link to a repository(ies) you recommend for Hackoberfest to the `repositories-list.md` file _only_ after the repository meets our requirements — check our [repository checklist](https://virtualcoffee.io/resources/developer-resources/open-source/maintainer-guide#repository-checklist). You can always return and add more repositories here.
+2. 📃 **Repositories List** — Add the link to a repository(ies) you recommend for Hacktoberfest to the `repositories-list.md` file _only_ after the repository meets our requirements — check our [repository checklist](https://virtualcoffee.io/resources/developer-resources/open-source/maintainer-guide#repository-checklist). You can always return and add more repositories here.
```
Comment (meg-gutshall) [inline CONTRIBUTING.md:28]: **suggestion (wording, non-blocking):** Reword slightly for clarity

```suggestion
2. 📃 **Repositories List** — Add the name and link to a repository you recommend for Hacktoberfest to the `repositories-list.md` file. Do this **_only after_** the repository meets our requirements — see our [repository checklist](https://virtualcoffee.io/resources/developer-resources/open-source/maintainer-guide#repository-checklist). You can add multiple repositories here!
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -10,33 +10,46 @@ Welcome to the contributing guide for Virtual Coffee's Preptember repo! Thank yo
...
+
+### 2. Fork this repository
+
+Click the `fork` button at the top of the front page.
 
-Click the `fork` button at the top of the front page. <br>
 Forking is a process to create a copy of this repository in your GitHub account. You always want to remember to fork a repository before cloning it.
 
-### 2. Clone the repository in your local machine
+### 3. Clone the repository in your local machine
```
Comment (meg-gutshall) [inline CONTRIBUTING.md:52]: **suggestion (non-blocking):** Abbreviate the heading

```suggestion
### 3. Clone the repository
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -66,33 +79,33 @@ Clone a repository means copying the remote repository into your local machine.
...
-### 3. Create a branch
+### 4. Create a branch
 
 A branch is an isolated environment to work on and save your changes. Later on, you will push this branch to the remote repository after you finish working on your changes.
 
 Run the following command to create a branch:
 
 ```bash
-git checkout -b working-branch-name
+git checkout -b BRANCH-NAME
```
Comment (meg-gutshall) [inline CONTRIBUTING.md:89]: **suggestion (non-blocking):** Make this match examples below

```suggestion
git checkout -b YOUR-BRANCH-NAME
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -179,17 +195,16 @@ Merge conflicts usually occur when changes are on the same line(s), in the same
...
 https://github.com/Virtual-Coffee/vc-preptember/assets/45172775/55a4e380-7b8b-4a36-a843-95237e66d4db
 
-You can always ask for help in the `#open-source` or `#help-and-pairing` channel in Slack. We're here to help you 💙!
-
-**Note**:
-
-- We recommend resolving the merge conflicts in your local environment rather than directly on GitHub.
-- For this contribution, we recommend you _accept both changes_.
+> [!TIP]
+> - You can always ask for help in the `#open-source` or `#help-and-pairing` channel in Slack. We're here to help you 💙!
```
Comment (meg-gutshall) [inline CONTRIBUTING.md:205]: **suggestion (non-blocking):**

```suggestion
> - You can always ask for help in the `#open-source` or `#help-and-pairing` channel in Slack. We're here to help you! 💙
```

File: repositories-list-2024.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+<!--
+✅ Relevant name
+✅ Description
+Clearly describe what the project does and is for.
+✅ Relevant tags
+They should highlight their scope, stack, field, etc.
+✅ README.md file
+Important information about your project.
+✅ CONTRIBUTIONS.md/CONTRIBUTING.md file
```
Comment (meg-gutshall) [inline repositories-list-2024.md:13]: **chore:** Change these to match PR template.

```suggestion
✅ README file
Important information about your project.
✅ CONTRIBUTING file
```

**Review by meg-gutshall — COMMENTED** [review]
I made some comments. Nothing is keeping me from hitting the merge button, mainly cleanup stuff.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -10,33 +10,46 @@ Welcome to the contributing guide for Virtual Coffee's Preptember repo! Thank yo
 
 We take our [Code of Conduct](https://virtualcoffee.io/code-of-conduct/) very seriously, and all contributors must abide by it. Let’s treat each other with respect, and remember, we are all people who care and are trying. If you believe someone has violated the Code of Conduct, please fill out our [CoC Violation Form](https://virtualcoffee.io/report-coc-violation/), which you can do anonymously.
 
-## New contributors
+## New Contributors
 
 If you're new to open source, check out the resources below to help you familiarize yourself with open source:
```
[the remark this answers is not in the document]
Comment (adiati98) [reply CONTRIBUTING.md:15]: As Git and GitHub are nouns, can we say like this?

```suggestion
If you're new to open source, check out the resources below to help you familiarize yourself with Git, GitHub, and the contributions flow:
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (adiati98) [reply CONTRIBUTING.md:28]: I really like this!

However, I want to also mention that after their first PR, folks may come back and create another PRs when they find more repos along the way.

Any idea on how to reword that?

Comment (dominicduffin1) [reply CONTRIBUTING.md:28]: @adiati98 @meg-gutshall How about this for the last sentence:

> You can add multiple repositories here and you can always make another PR if you have more for us later in the month!

File: CONTRIBUTING.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (dominicduffin1) [reply CONTRIBUTING.md:15]: @adiati98 I'd be happy with your update for line 15, but (minor grammar point) I think it should be "the contribution flow" without he plural.

**Comment by adiati98** [comment]
@meg-gutshall & @dominicduffin1,

I've made the changes.
I've also adjusted the format in the "Contribution Type" to make us easier in updating the theme (favorite thing) for our next Preptembers. 

Considering that today is 1 September already, can any of you approve and merge this PR?
If we need to fix anything else, we can always do it later.

Thanks so much!

**Review by dominicduffin1 — APPROVED** [review]

**Comment by dominicduffin1** [comment]
Thank you @adiati98

**Pull request merged**